src/common/image.cpp

```diff
@@ -157,10 +164,10 @@ bool Image::resize(int newW, int newH) {
 
 VTFImageFormat Image::vtf_format() const {
 	switch (m_type) {
-		case imglib::UInt16:
-			// @TODO: How to handle RGBA16? DONT i guess
-			return IMAGE_FORMAT_RGBA16161616F;
-		case imglib::Float:
+		case ChannelType::UInt16:
+			// @TODO: How to handle RGB16? DONT i guess
```

Comment:
RGB161616 isn't a valid format (at least for VTFs)

Reply:
Yes, that's what the comment is indicating. I considered adding RGB161616 to VTFLib to work around this.